src/neptune/utils.py

```diff
@@ -46,11 +46,16 @@
 logger = get_logger()
 
 
-def stringify_unsupported(value: Any) -> Union[StringifyValue, Mapping]:
+def stringify_unsupported(
+    value: Any,
+    expand: bool = False,
+) -> Union[StringifyValue, Mapping]:
     """Helper function that converts unsupported values in a collection or dictionary to strings.
 
     Args:
         value (Any): A dictionary with values or a collection
+        expand (bool, optional): If True, the function will expand series to store each item as a key-value pair.
```

Comment:
I think a more in-depth explanation is necessary. Perhaps an additional example? cc @normandy7